Tests/Integration/Network/Protocol/ConnectionIntTests.swift

```diff
@@ -7,85 +7,66 @@ import LibMobileCoin
 import XCTest
 
 class ConnectionIntTests: XCTestCase {
-    func testCallTimeoutWithInvalidUrl() throws {
+
+    func testInvalidUrlTimeout() throws {
         try TransportProtocol.supportedProtocols.forEach { transportProtocol in
-            try testCallTimeoutWithInvalidUrl(transportProtocol: transportProtocol)
+            print("Testing \(transportProtocol.description) with timeout = " +
+                  "\(transportProtocol.timeoutInSeconds)")
+            try testInvalidUrlTimeout(using: transportProtocol)
         }
     }
 
-    func testCallTimeoutWithInvalidUrl(transportProtocol: TransportProtocol) throws {
+    func testInvalidUrlTimeout(using transportProtocol: TransportProtocol) throws {
         let expect = expectation(description: "Making Fog View enclave request")
 
         var request = FogLedger_BlockRequest()
         request.rangeValues = [1..<2]
-        try createFogBlockConnectionWithInvalidUrls(transportProtocol: transportProtocol).getBlocks(request: request) {
-            guard let error = $0.failureOrFulfill(expectation: expect) else { return }
+        try IntegrationTestFixtures.createFogBlockConnectionWithInvalidUrls(
```

Comment:
fixture pattern is not being properly implemented, can you look at some of the other test files in the project ? The new ones Ive added for memos come to mind, but alot of the other ones use this pattern. If you want to pair up on one of these I could do that.